.travis.yml

```diff
@@ -2,14 +2,17 @@ language: node_js
 node_js:
   - "stable"
 sudo: false
-env: REACT_VERSION=15.3.x
+env: REACT_VERSION=15.5.x
```

Comment:
another way to do this is to use a `MAIN_RUN` variable, so you don't have to duplicate what's in package.json. Nbd either way tho. Example here: https://github.com/brigade/react-waypoint/blob/be503a009091eb6c0522419a007fb135c15ce92f/.travis.yml#L9-L10